# How far the trajectory should be moved to the left (positive offset).
# A negative offset moves the trajectory to the right.
float32 offset
# If the overtake should start at start_pos. If False, will start the overtake immediately.
bool has_start_pos
# Global start position
geometry_msgs/Point start_pos
# If the overtake should end at end_pos. If False, will stay in the overtake indefinitely
# until an EndOvertake request is received
bool has_end_pos
# Global end position
geometry_msgs/Point end_pos

# Transition length towards the overtake
float32 start_transition_length
# Transition length away from the overtake
float32 end_transition_length

---

string msg
bool success
